docs/refs/catalog-queries.md

```diff
@@ -0,0 +1,41 @@
+# Catalog queries
+
+You can use the `curl` command with `jq` to query catalogs that are installed on your cluster.
+
+``` terminal title="Query syntax"
+$ curl http://localhost:8080/catalogs/operatorhubio/all.json | <query>
+```
+
+## Package queries
+
+Available packages in a catalog
+: `jq -s '.[] | select( .schema == "olm.package")`
```

Comment:
Nit: For all of the `jq` commands shown here can we use the "code fence" syntax (i.e triple backtick)? This results in a user being able to easily copy the command to their clipboard via a handy dandy button and IMO presents a nicer docs experience when used where possible.

Reply:
I created a follow up issue: https://github.com/operator-framework/operator-controller/issues/698

Basically, I can't get the code blocks to render in the definition list. I think I might need to move the list to a table, but if I get the code block to work, the table gets stretched really wide. 

In the interest of making the Kubecon deadline, I would prefer to get the rest of the content in place and circle back after. WDYT, @everettraven?

Reply:
SGTM - I don't think this nit should at all block this from merging!